.github/workflows/autopep8.yml

```
@@ -0,0 +1,23 @@
+name: autopep8
```

```suggestion
name: "Code Format"
```
Python以外ののフォーマットチェックも追加しやすいように、名前は一般的なものにしておこう